assets/scss/base/_lists.scss

```diff
@@ -14,6 +25,26 @@ ul, nav ul {
 	}
 }
 
+/*
+Numbered list
+
+An ordered list with numbers where the parent element uses the modifier `.list--numbered` and the child elements use the class `.numbered`
+
+Markup:
```

Comment:
You won't have access to all orgs repos though. @feedmypixel can you clarify this?